### DISCUSSION TITLE
jetcd-0.7.4 Tag

### DISCUSSION THREAD

**Discussion by darvay** [etcd-io/jetcd#1110] [General]
Hope all is well, when can the 0.7.4 tag be cut for the latest release. Thanks in advance, David

**Comment by lburgazzoli** [comment]
I'll try to cut something before end of the week

**Reply by darvay** [reply]
Thanks @lburgazzoli !

**Comment by lburgazzoli** [comment]
jetcd 0.7.5 is on [maven central](https://repo1.maven.org/maven2/io/etcd/jetcd-core/0.7.5)
version 0.7.4 has been skipped because of a release error
